staging/src/k8s.io/controller-manager/options/generic.go

```diff
@@ -58,6 +58,7 @@ func (o *GenericControllerManagerConfigurationOptions) AddFlags(fss *cliflag.Nam
 	o.LeaderMigration.AddFlags(fss.FlagSet("leader-migration"))
 	genericfs := fss.FlagSet("generic")
 	genericfs.DurationVar(&o.MinResyncPeriod.Duration, "min-resync-period", o.MinResyncPeriod.Duration, "The resync period in reflectors will be random between MinResyncPeriod and 2*MinResyncPeriod.")
+	genericfs.StringVar(&o.ClientConnection.Kubeconfig, "kubeconfig", o.ClientConnection.Kubeconfig, "Path to kubeconfig file with authorization and master location information (the master location can be overridden by the master flag).")
```

Comment:
binding this flag here referencing a master flag that doesn't exist in this layer doesn't make much sense... can we keep the flag bindings where they are but bind to `s.Generic.ClientConnection.Kubeconfig` instead?

Reply:
Thanks @liggitt 
That does make more sense.
Updated.